api/README.md

```diff
@@ -3,7 +3,7 @@
 
 ## Requirements
 
-- python (3.x)
+- python (>2.7,<=3.8)
```

Comment:
why add the `>2.7` ?

Reply:
Because from the begining we were not supporting that version 🤔